features/rni.feature

```diff
@@ -0,0 +1,202 @@
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | naam.voornamen                  |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 101                                             |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Persoon                                         |
+
+    Scenario: RNI-deelnemers op geleverde gegevens uit meerdere categorieën
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000024                                   |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                            | waarde                               |
+      | land (13.10)                    | 5010                                 |
+      | RNI-deelnemer (08.88.10)        | 201                                  |
+      | Omschrijving verdrag (08.88.20) | Artikel 45 EU-Werkingsverdrag (VWEU) |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                |
+      | type                | RaadpleegMetBurgerservicenummer                       |
+      | burgerservicenummer | 000000024                                             |
+      | fields              | naam.voornamen,verblijfplaats.verblijfadres.land.code |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En heeft de persoon de volgende 'verblijfplaats' gegevens
```

Comment:
`En heeft de persoon een 'verblijfplaats' met de volgende gegevens

Reply:
> `En heeft de persoon een 'verblijfplaats' met de volgende gegevens`

Deze stap moet alleen worden gebruikt bij gegevensgroepen die meerdere keren kunnen voorkomen bij een persoon, zoals kinderen, partners, ouders en nationaliteiten

Reply:
Dus "En heeft de persoon de volgende 'verblijfplaats' gegevens" is goed, @MelvLee?

Reply:
De huidige step definitie blijkt bij nader inzien ook prima te werken.